src/TLuaInterpreter.cpp

```diff
@@ -820,29 +815,27 @@ int TLuaInterpreter::wrapLine(lua_State* L)
 {
     int s = 1;
```

Comment:
*Slightly* more efficient to set `s` to `0` here and then increment it in `if (!lua_is`{`string`|`number`}`(L, ++s)) {` calls.

Reply:
Yes - but I'm not up for doing a ton of testing right now, so I'll leave it be